src/app/drag-and-drop/grouping.demo.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component} from "@angular/core";
+import {ClrDragEvent} from "../../clr-angular/utils/drag-and-drop/drag-event";
+
+const FILES = [
+    {name: "img_001.jpg", group: "image-file"}, {name: "mov_001.mov", group: "movie-file"},
+    {name: "aud_001.mp4", group: "audio-file"}
+];
+
+@Component(
+    {selector: "basic-droppable-demo", styleUrls: ["./drag-and-drop.demo.scss"], templateUrl: "./grouping.demo.html"})
```

Comment:
formatting?

Reply:
This is what our old clang format is doing for some reason.